### ISSUE TITLE
Elytra movement broken when falling though leaves.

### ISSUE THREAD

**Issue by badcrave** [dexman545/Fabric-Passable-Leaves#3]
When elytra are equipped and the player is in flight mode and the player is passing through leaves, it will behave as though the player is taking damage but the damage (usually) doesn't decrease. It will override the elytra movement with falling through leaves, but slightly slower.

**Comment by dexman545** [comment]
It is intended for flight through leaves to be slowed and damage to be taken, damage dealt is relative to velocity. In some cases armor will prevent loss of health.

Is flight being entirely canceled (no forward movement)?

**Comment by badcrave** [comment]
Yeah kinda, if you dont have enough momentum. Is there a way it could be disabled? Or preferably the damage would be disabled?

**Comment by dexman545** [comment]
Currently not possible, could be added.
